README.md

```diff
@@ -51,11 +51,11 @@ In Linux, these can all be acquired at the same time with the following command:
   - Change to the `$MENGE_ROOT/projects/g++` directory
   - type: `make`
   
-It will attempt to build both Menge and its Plug-ins.  It will not attempt to build the
+It will build both Menge and its Plug-ins.  It will not build the
```

Comment:
Removing "attempt to" suggests a fool-proof system.  It's not a guarantee I'm willing to make.